samples/sampler/stories/default/WizardPanels.js

```diff
@@ -1,15 +1,17 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
```

Comment:
Could you fix qa-sampler too?
![image](https://user-images.githubusercontent.com/34676178/117264512-3f912d00-ae8e-11eb-81de-d2a187b198dc.png)